pegasus-spark-analyser/src/main/java/com/xiaomi/infra/pegasus/spark/analyser/Config.java

```diff
@@ -0,0 +1,12 @@
+package com.xiaomi.infra.pegasus.spark.analyser;
+
+public interface Config {
+
+  enum DataType {
+    COLD_BACKUP,
+    ONLINE,
+    INVALID
+  }
+
+  DataType getDataType();
```

Comment:
From common design principles, the interface should have no knowledge of the concrete type of subclasses.
I am for now not sure if this design is good. Just give notice here.

Reply:
`interface` just offer common method which is overloded and used by subclasses. `interface` can't be instanced, so it also can't use the abstract method